### PR TITLE
feat: submit a review comment

### PR DESCRIPTION
**Pull request by Ece-Kahraman**

### CODE REVIEW

File: api-server/Web/Controllers/GitHubController.cs [hunk 1]
```diff
@@ -640,6 +644,15 @@ public async Task<ActionResult> getRevCommentsOnPR( string owner, string repoNam
...
 
+    /*[HttpPost("pullrequest/{owner}/{repoName}/{prnumber}/addRevComment")]
+    public async Task<ActionResult> AddRevCommentToPR(string owner, string repoName, int prnumber, [FromBody] string commentBody)
+    {
+        var client = GetNewClient(_httpContextAccessor?.HttpContext?.Session.GetString("AccessToken"));
+        PullRequestRe
+        await client.PullRequest.ReviewComment.Create()  Issue.Comment.Create(owner, repoName, prnumber, commentBody);
+        return Ok($"Comment added to pull request #{prnumber} in repository {repoName}.");   
+    }*/
+
```
Comment (vIremAydin) [inline api-server/Web/Controllers/GitHubController.cs:655]: delete the comments please

File: api-server/Web/Controllers/GitHubController.cs [hunk 2]
```diff
@@ -738,19 +752,18 @@ public async Task<ActionResult> getRepositoryContributors(string owner, string r
...
-                        if (contributor.Login == userLogin)
-                            continue; // Skip the logged-in user
+                        if (collaborator.Login == prOwner)
+                            continue; // Skip the pr owner
                         
-                        result.Add(new ContributorInfo
+                        result.Add(new CollaboratorInfo
                         {
-                            Id = contributor.Id,
-                            Login = contributor.Login,
```
Comment (vIremAydin) [inline api-server/Web/Controllers/GitHubController.cs:751]: use "name" instead of login

**Comment by vIremAydin** [comment]
there are some minor problems

**Comment by aysekelleci** [comment]
<p>test <em>comment </em></p><p><strong><em>dwefrght </em></strong></p><p><strong><em><u>dfegrhtjy</u></em></strong></p>

**Comment by aysekelleci** [comment]
<p>test</p>

**Pull request merged**